nservicebus/hosting/azure-functions/service-bus_queue-trigger-wiring_ASBFunctions_[1.2,).partial.md

```diff
@@ -1,19 +0,0 @@
-The Azure Function trigger for NServiceBus is auto-generated by specifying the logical endpoint name using a custom assembly attribute:
-
-snippet: endpoint-trigger-function-wireup
-
-The attribute will generate the trigger function required for NServiceBus:
-
-snippet: asb-function-hostbuilder-trigger
-
-Note: An invalid endpoint name will generate an `NSBFUNC001` error with the message `Endpoint name is invalid and cannot be used to generate trigger function`.
-
-If the trigger function needs to be customized, the trigger function generation can be disabled by removing the `NServiceBusEndpointName` attribute. A customized trigger function can then be manually added to the project.
-
-#### Overriding the trigger function name
-
-The trigger function name is auto-generated by default. To customize the function name, the `NServiceBusEndpointName` attribute can be provided with an additional parameter to set the function name:
-
-snippet: asb-function-override-trigger-function-name
-
-Note: An invalid trigger function name will generate an `NSBFUNC002` error with the message `Trigger function name is invalid and cannot be used to generate trigger function`.
```

Comment:
Why is this removed? Are these snippets no longer used? When I render this locally, the 1.2 page has a heading with no content for this:

![image](https://user-images.githubusercontent.com/124014/128974584-387518d3-5c4f-4456-abc2-4452aa2530b0.png)

Reply:
fixed, the md file was deleted causing issues.